src/NServiceBus.Testing/TestingContext.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace NServiceBus.Testing
+{
+    using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using NServiceBus.Testing.ExpectedInvocations;
+
+    class TestingContext : TestableMessageHandlerContext
+    {
+        public TestingContext(IMessageCreator messageCreator = null) : base(messageCreator)
+        {
+        }
+
+        public TestingContext(IMessageCreator messageCreator, TimeoutMessage<object>[] timeoutMessages) : base(messageCreator)
+        {
+            this.timeoutMessages = new ConcurrentQueue<TimeoutMessage<object>>(timeoutMessages);
+        }
+
+        public TimeoutMessage<object>[] TimeoutMessages => timeoutMessages.ToArray();
+
+        internal IList<ExpectInvocation> ExpectedInvocations { get; } = new List<ExpectInvocation>();
```

Comment:
since the class is internal, should this just be public? Or address my comment from https://github.com/Particular/NServiceBus.Testing/pull/67/files#r56180296